gcp-go-functions/README.md

```diff
@@ -0,0 +1,52 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Google Cloud Functions in Python deployed with Go
+
+This example deploys a Google Cloud Function implemented in Python. Pulumi program is implemented in Go.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+1. [Configure Pulumi for GCP](https://www.pulumi.com/docs/intro/cloud-providers/gcp/setup/)
+    - [Set up a Service Account](https://www.pulumi.com/docs/intro/cloud-providers/gcp/service-account/)
+
+### Steps
+
+1.  Build `main.go`:
+
+    ```bash
+    go build -o gcp-go-functions main.go
+    ```
```

Comment:
I believe this step is not needed - right @EvanBoyle ?  I think if it's not necessary we should skip including it by default.

Reply:
Apparently so: https://github.com/pulumi/examples/pull/553#discussion_r379110361. I'll delete this step.